### ISSUE TITLE
Problem with umlauts (ä, ö ,ü, ...)

### ISSUE THREAD

**Issue by cyberius0** [mjeanroy/exiftool#405]
Hello,

I am having problems, when my files have umlauts (ä, ö ,ü, ..) in their filenames. Then `exifTool.getImageMeta` returns nothing.
No error message is shown.

I am using Windows 10.
Java 22.
ExifTool Version Number: 12.89

For example, this code
```
System.out.println( System.getProperty( "file.encoding" ) );

System.out.println(exifTool.getImageMeta(new File("C:\\1ä.jpg"), Arrays.asList(StandardTag.MODEL)));

System.out.println(exifTool.getImageMeta(new File("C:\\2.jpg"), Arrays.asList(StandardTag.MODEL)));
```

gives this output:
```
UTF-8
{}
{MODEL=Pixel 7a}
```

(Yes, both images contain the MODEL tag).
Parsing those files directly with exiftool.exe is working.
What can I do?

**Comment by mjeanroy** [comment]
Hi,
Could you give me more information:
- What jdk version?
- What operating system? Windows/Linux/MacOS?

I just pushed a unit test with a file containing an umlaut char, and it looks like it's only failing on Windows with a jdk >= 19 (see [here](https://github.com/mjeanroy/exiftool/actions/runs/9942877526/job/27466083732)), is it consistent with your setup?

**Comment by mjeanroy** [comment]
Hi @cyberius0,

I found the issue (due to encoding difference between Windows and the rest of the world).
It should be fixed with version 3.0.1 (that has just been released).

Let me know if it works for you or if you still have some issues!

**Comment by cyberius0** [comment]
@mjeanroy Thank you very much !!
Version 3.0.1 fixed this problem.
